dockers/ubuntu-cuda/Dockerfile

```diff
@@ -136,7 +136,14 @@ RUN \
         cd .. && \
         rm -rf Fuser ; \
     elif [ "${TORCH_INSTALL}" == "test" ]; then \
-        echo "Not supported option" ; \
+        # building nvFuser from source
```

Comment:
@mruberry, which version of nvFuser shall we use for the test version of Torch 2.3
for PT 2.2 there is PyPI version but none was yet build for `nvfuser-cu118-torch23`

Reply:
I think this is actually more a @jjsjann123 question

Reply:
I think it is fine for now to compile it from source...